### PR TITLE
Collect both lister components in one place

### PR DESCRIPTION
**Pull request by virresh**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/itemTemplates/card.html [hunk 1]
```diff
@@ -11,9 +11,12 @@ <h3 class="cmp-portallister__item--card-title" data-cmp-hook-item-template="item
             <p class="cmp-portallister__item--card-description" data-cmp-hook-item-template="description"></p>
         </div>
         <div class="cmp-portallister__item--card-footer">
-            <div class="cmp-portallister__item--card-operations" data-cmp-hook-item-template="operations">
+            <div class="cmp-portallister__item--card-operations" aria-label="Menu" data-cmp-hook-item-template="operations">
```
Comment (sakshi-arora1) [inline ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/itemTemplates/card.html:14]: should be actions menu... or take doc team input

File: ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/clientlibs/layouts/iteminjector/js/itemapi.js [hunk 2]
```diff
@@ -152,6 +152,11 @@
                 linkElem.setAttribute("link", data.formLink);
                 linkElem.classList.add('item-link');
                 linkElem.addEventListener("click", linkClickHandler);
+//            if (linkElem) {
```
Comment (sakshi-arora1) [inline ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/clientlibs/layouts/iteminjector/js/itemapi.js:155]: remove the commented lines

File: ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/itemTemplates/card.html [hunk 3]
```diff
@@ -11,9 +11,12 @@ <h3 class="cmp-portallister__item--card-title" data-cmp-hook-item-template="item
...
                 <div class="cmp-portallister__item--card-operations-icon"></div>
             </div>
         </div>
+<!--/**        <img class="cmp-portallister__item&#45;&#45;card-image" data-cmp-hook-item-template="thumbnail"/>*/-->
```
Comment (sakshi-arora1) [inline ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/itemTemplates/card.html]: remove the commented lines

File: bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/formsportal/DraftsAndSubmissionsImpl.java [hunk 4]
```diff
@@ -180,9 +167,20 @@ public String getOperationName() {
...
         List<PortalLister.Item> itemList = new ArrayList<>();
         TypeEnum typeEnum = TypeEnum.valueOf(getType());
+
         QueryImpl query = new QueryImpl();
         query.setOffset(getOffset());
         query.setLimit(getLimit());
+
+        // Add information about currently logged in user
+        StatementImpl currentUserStatement = new StatementImpl();
+        currentUserStatement.setAttributeName("owner");
```
Comment (sakshi-arora1) [inline bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/formsportal/DraftsAndSubmissionsImpl.java:177]: this is being honoured in portal API?

Comment (virresh) [reply bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/formsportal/DraftsAndSubmissionsImpl.java:177]: Yes, as per discussion with Ratish
Still have to test it out with a login component

**Comment by virresh** [comment]
Opening pull request in upstream, closing PR on fork.

**Pull request closed without merging**